### PR TITLE
[Mono] Enable runtime tests running on arm64 with mini JIT

### PR DESCRIPTION
**Pull request by fanyang-mono**

### CODE REVIEW

File: src/tests/issues.targets [hunk 1]
```diff
@@ -3271,6 +3271,87 @@
         </ExcludeList>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'mono' and '$(RuntimeVariant)' == 'minijit' and '$(TargetArchitecture)' == 'arm64'">
+        <ExcludeList Include="$(XunitTestBinBase)/Interop/SuppressGCTransition/SuppressGCTransitionTest/**">
+            <Issue>needs triage</Issue>
```
Comment (SamMonoRT) [inline src/tests/issues.targets:3276]: For this and below, do we know why these are failing to add GH issues for each? If not, to get them tracked, add all failing tests to one tracking issue and add that GH issue# here.

Comment (fanyang-mono) [reply src/tests/issues.targets:3276]: Will create an issue.

**Comment by steveisok** [comment]
> /cc @steveisok - this is a new lane being stood up to cover newer intrinsic work we are doing for JIT. Not sure if this is something we need to run by anyone and determine what loop it runs in?

I don't think there are any restrictions if it runs like the others.

**Review by jandupej — APPROVED** [review]

**Review by LeVladIonescu — APPROVED** [review]

**Comment by fanyang-mono** [comment]
The new lane added by this PR `runtime (Build linux-arm64 Release AllSubsets_Mono_Minijit_RuntimeTests minijit)` has passed the testing.

**Comment by fanyang-mono** [comment]
CI failures are not related to the change in this PR.

**Pull request merged**